AtlasUI.py

```diff
@@ -116,13 +118,12 @@ def _webhandle_data_GET(self):
                 'mode':                self.simEngine.mode(),
                 'simulatedTime':       self.simEngine.formatSimulatedTime(),
                 'dotBots':             returnValDotBots,
-                'discomap':            orchestratorView['discomap'],
```

Comment:
you're right to remove this, but are you sure you have remove all discomap related code everywhere, in particular in the GUI?